setup-scripts/serverload.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+: '
+This script writes the CPU usage and the free disk space to a file in a loop.
+The results are prepended to the file, so the most recent results are at the
+top.  This is so that the most recent information can easily be read from the
+top of the file and the file can easily be truncated if needed.
+'
+
+# Define Loop Interval
+loop_interval=60
+webportal_repo_setup_scripts="/home/user/skynet-webportal/setup-scripts"
+logfile_name="serverload.log"
+logfile=$webportal_repo_setup_scripts/$logfile_name
+jsonfile="serverload.json"
+nginx_docker_path="/usr/local/share"
+
+# Create logfile if it doesn't exist
+if [[ ! -e $logfile ]]; then
+    echo "init" > $logfile 
+fi
+
+# Write the output in an infinite loop.
+while true; do
+    # CPU usage
+    cpu=$(grep 'cpu ' /proc/stat | awk '{usage=($2+$4)*100/($2+$4+$5)} END {print usage "%"}') 
```

Comment:
I'm not sure whether this command is what we want. I think you got this from [stacko](https://stackoverflow.com/questions/9229333/how-to-get-overall-cpu-usage-e-g-57-on-linux) right?

There is a note on that thread that says:
> NOTE: this is not the current CPU usage, but the overall CPU usage in all the cores since the system bootup. This could be very different from the current CPU usage. To get the current value top (or similar tool) must be used.

It goes on to suggest:
> awk '{u=$2+$4; t=$2+$4+$5; if (NR==1){u1=u; t1=t;} else print ($2+$4-u1) * 100 / (t-t1) "%"; }' \
<(grep 'cpu ' /proc/stat) <(sleep 1;grep 'cpu ' /proc/stat)

The current command I believe will give the average CPU load since the server was started, 
so it would be a (hugely) lagging indicator. Perhaps we should use something like `top` for this instead.

Reply:
I just got the command from David. Easy enough to update to whatever we want.

Reply:
```suggestion
    cpu=$(echo $[100-$(vmstat 1 2|tail -1|awk '{print $15}')]) 
```

Reply:
This is an updated command from David. We can indeed change it whenever so I decided to apply that one and resolve your comment so we can merge this today.